recipes/pbwt/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: pbwt
+  version: "3.0"
+
+source:
+  # Using a specific git checkout of 0.98 because of a minor fix in the Makefile
+  url: https://github.com/richarddurbin/pbwt/archive/64c4ffa7880db5f85b53ccbd7e530ec4b95ea4ba.zip
+  sha256: 57b02f4ec56d236a3f4e6e9789dbefbc505a3157840edcbee2201a831806003e
+
+build:
+  number: 0
+  skip: true # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - htslib
+    - zlib
+  run:
+   - htslib
+   - zlib
+
+test:
+  commands:
+    - pbwt > /dev/null
+
+
+about:
+  home: https://github.com/richarddurbin/pbwt
+  license: GPL
```

Comment:
can you please add license_file and license_family ... if it is GPL we need to ship a license_file ... strictly ... or just ignore me ;)

Reply:
I actually had the license wrong :(